Library/Formula/openshift-cli.rb

```diff
@@ -0,0 +1,36 @@
+class OpenshiftCli < Formula
+  desc "OpenShift command-line interface tools"
+  homepage "https://www.openshift.com/"
+  url "https://github.com/openshift/origin.git",
+    :tag => "v1.1.1",
+    :revision => "e1d9873c1d5711b83fd3dd7eefe83a88ceb92c08"
+
+  head "https://github.com/openshift/origin.git"
+
+  depends_on "go" => :build
+
+  def install
+    # this is necessary to avoid having the version marked as dirty
+    ignore_file = buildpath/".git/info/exclude"
+    ignore_file.open("w") do |file|
+      file.puts "/.brew_home"
+    end
+
+    system "make", "all", "WHAT=cmd/openshift", "GOFLAGS=-v"
```

Comment:
Yes that's correct.

On Tue, Jan 19, 2016 at 1:19 AM, Eli Young <notifications@github.com> wrote:

> In Library/Formula/openshift-cli.rb
> <https://github.com/Homebrew/homebrew/pull/48231#discussion_r50076126>:
>
> > +  url "https://github.com/openshift/origin.git",
> > +    :tag => "v1.1.1",
> > +    :revision => "e1d9873c1d5711b83fd3dd7eefe83a88ceb92c08"
> > +
> > +  head "https://github.com/openshift/origin.git"
> > +
> > +  depends_on "go" => :build
> > +
> > +  def install
> > +    # this is necessary to avoid having the version marked as dirty
> > +    ignore_file = buildpath/".git/info/exclude"
> > +    ignore_file.open("w") do |file|
> > +      file.puts "/.brew_home"
> > +    end
> > +
> > +    system "make", "all", "WHAT=cmd/openshift", "GOFLAGS=-v"
>
> When that happens, we'd change this line to system "make", "all",
> "WHAT=cmd/oc", "GOFLAGS=-v", correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/48231/files#r50076126>.
>